test/test_cabin_helper.rb

```diff
@@ -0,0 +1,199 @@
+$: << File.dirname(__FILE__)
+$: << File.join(File.dirname(__FILE__), "..", "lib")
+
+require "rubygems"
+require "minitest-patch"
+require "cabin"
+
+module RSpec
+  class Confi
```

Comment:
I'll go through soon and rewrite the tests to  use rspec instead of minitest, and then this concern becomes resolved :)